compiler/src/dmd/lexer.d

```diff
@@ -2393,6 +2439,47 @@ class Lexer
         }
     }
 
+    /**************************************
+    */
+    private Identifier acquireIdentifierFromString(scope const(char)[] s, bool isNormalized)
```

Comment:
No documentation as to what this function does.

Reply:
This is another of the functions I need feedback on.

It applies the strategy of normalization wrt. warnings/errors ext. before interning.
I had to split it out due to two paths ways calling into it.

We could move it into ``idPoolNormalize`` but that would require passing more information in.
We should probably come back to it after strategies have been decided upon.

But yes, needs documentation.

Reply:
Documented.

Reply:
> It applies the strategy of normalization wrt. warnings/errors ext. before interning.
I had to split it out due to two paths ways calling into it.

I don't know what that means, or how that applies to isNormalized.

Reply:
A universal character name (ImportC only) has to branch and do some extra memory allocation before creating an identifier.

I couldn't figure out how to unify the two branches so that the identifier creation only happened after it.

Basically, we call ``idPoolNormalize`` rather than ``idPool`` to handle unnormalized identifiers.